presto-verifier/src/main/java/com/facebook/presto/verifier/rewrite/QueryRewriter.java

```diff
@@ -203,21 +211,29 @@ public QueryObjectBundle rewriteQuery(@Language("SQL") String query, ClusterType
                         REWRITE,
                         SHOW_CREATE_VIEW_CONVERTER).getResults());
                 CreateView createExistingView = (CreateView) sqlParser.createStatement(createExistingViewQuery, PARSING_OPTIONS);
+                Query viewQuery = createExistingView.getQuery();
```

Comment:
Insert and create I understand - we do if for verifier.
What are the view queries?

Reply:
I can change to skip view, as we probably don't have it in prod traffic.

As you might know, CreateView is like define a virtual view, and the view name is simply replaced with the original sql during execution.